bikeshed/__init__.py

```diff
@@ -1502,7 +1502,7 @@ def idlReplacer(match):
             return E.code({"class":"idl"},
                 E.a({"data-link-type":linkType, "for": match.group(1)}, match.group(2)))
 
-        elementRe = re.compile(r"<{(?:(\w+)/)?(\w+)}>")
+        elementRe = re.compile(r"<{(?:([^ \"'>/=]+)/)?([^ \"'>/=]+)}>")
```

Comment:
Can it? It would stop after hitting a `/` or `>`, right?

But whatever. :) `\w-` is probably clearer.